account_move_report/views/account_move_report.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="account_entries_report" name="Entries Report">
+        <t t-call="web.external_layout">
+            <t t-set="count" t-value="0"/>
+            <t t-set="count_total" t-value="len(docs)"/>
+            <t t-foreach="docs" t-as="o">
+                <t t-set="count" t-value="count+1"/>
```

Comment:
There is no need to define a variable to store the loop index. You may use the predefined valiable `variablename_index`. In this case, it would be `o_index`.

For more info, you may check the documentation:  
https://www.odoo.com/documentation/12.0/reference/qweb.html#loops